src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -58,19 +58,19 @@ public void testDot()
         checkConfig.addAttribute("option", "NL");
         checkConfig.addAttribute("tokens", "DOT");
         final String[] expected = {
-            "31:10: " + getCheckMessage(MSG_LINE_NEW, "."),
+            "29:10: " + getCheckMessage(MSG_LINE_NEW, "."),
```

Comment:
[Issue says `copy`](https://github.com/checkstyle/checkstyle/issues/3904#issue-211755288). I think we should stick with copy as trimming input requires reviewing what was changed to make sure something necessary wasn't accidentally removed. Violation cases are hard to miss, but good cases are easy to miss as there is no verification in test. Since file is copied and then changed in 1 commit, we can't see in history what was changed.
@MEZk I am fine with trimming input, but it should be done outside this issue and most likely after the parent issue for this is finished as we have alot of tests to go through.
@romani can override this, but I stand by that we should only be doing copies for now since we are overloaded with PRs and GSoC.